app/src/main/kotlin/org/fossify/phone/adapters/ConferenceCallsAdapter.kt

```diff
@@ -50,11 +50,12 @@ class ConferenceCallsAdapter(
                 getCallContact(itemView.context, call) { callContact ->
                     root.post {
                         itemConferenceCallName.text = callContact.name.ifEmpty { itemView.context.getString(R.string.unknown_caller) }
+                        itemConferenceCallName.setTextColor(textColor)
                         SimpleContactsHelper(activity).loadContactImage(
                             callContact.photoUri,
                             itemConferenceCallImage,
                             callContact.name,
-                            activity.getDrawable(R.drawable.ic_person_vector)
+                            activity.getDrawable(R.drawable.ic_person_vector)?.apply { setTint(textColor) }
```

Comment:
Let's call `mutate()` before changing the color. There's a helper for function for this:

```suggestion
                            activity.getDrawable(R.drawable.ic_person_vector)?.applyColorFilter(textColor)
```